cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -513,7 +522,23 @@ teeForge ft tverb tr = Tracer $ \ev -> do
       Consensus.TraceBlockFromFuture{} -> teeForge' (ftTraceBlockFromFuture ft)
       Consensus.TraceSlotIsImmutable{} -> teeForge' (ftTraceSlotIsImmutable ft)
       Consensus.TraceNodeIsLeader{} -> teeForge' (ftTraceNodeIsLeader ft)
-  traceWith (toLogObject' tverb tr) ev
+  case event of
+    Consensus.TraceStartLeadershipCheck slot -> do
+      !utxoSize <- mapNodeKernelDataIO nkUtxoSize nodeKern
+      meta <- mkLOMeta sev Public
+      traceNamedObject tr
+        ( meta
+        , LogStructured $ Map.fromList $
+          [("kind", String "TraceStartLeadershipCheck")
+          ,("slot", toJSON $ unSlotNo slot)]
+          ++ fromSMaybe [] ((:[]) . (Text.pack "utxoSize",) . toJSON <$> utxoSize))
```

Comment:
The `OverloadedSrtings` pragma is enabled so the `Text.pack` is superfluous.

Reply:
Done!